resources/tools/strip-emoji/index.js

```diff
@@ -0,0 +1,18 @@
+
+const emojiRegex = require('emoji-regex');
+
+process.stdin.setEncoding('utf-8');
+
+// We will concatenate all strings and assume we are not processing a huge file,
+// so we don't break in the middle of a UTF-8 sequence or split emoji sequences
+// in half.
```

Comment:
```suggestion
// We will concatenate all strings and assume we are not processing a huge file,
// so we don't split emoji sequences in half.
```

TIL that setEncoding will ensure that UTF-8 is not broken. But I'm assuming that Node will potentially split multi-codepoint emoji sequences...